Makefile.am

```diff
@@ -213,13 +215,27 @@ cpp_server_ct_mirror_LDADD = \
 	$(leveldb_LIBS) \
 	-lprotobuf -lsqlite3
 cpp_server_ct_mirror_SOURCES = \
-  cpp/client/async_log_client.cc \
+        cpp/client/async_log_client.cc \
 	cpp/server/ct-mirror.cc \
 	cpp/server/certificate_handler.cc \
 	cpp/server/handler.cc \
 	cpp/server/json_output.cc \
 	cpp/server/server_helper.cc
 
+cpp_server_ct_mirror_v2_LDADD = \
+	cpp/libcore.a \
+	$(json_c_LIBS) \
+	$(libevent_LIBS) \
+	$(leveldb_LIBS) \
+	-lprotobuf -lsqlite3
+cpp_server_ct_mirror_v2_SOURCES = \
+        cpp/client/async_log_client.cc \
```

Comment:
I give up, just look at it on GitHub, it's all fairly clear there, somehow? `cpp_server_xjson_server_SOURCES` below looks totally wacky before this PR, even (your one change to it seems to be for the better, but the other lines are all randomly indented?).

Good luck. :smiley: